app/models/concerns/solr_document_extensions/asset.rb

```diff
@@ -77,4 +77,20 @@ def publishable?
   def caption
     Array(self[Solrizer.solr_name('caption', :stored_searchable)]).first
   end
+
```

Comment:
Just curious @awead, what does this do?

Reply:
@RudyOnRails these are methods on SolrDocument fields that are specific to assets. It's just a way of grouping code.